checkpoint/cp-resource-management/README.md

```diff
@@ -16,21 +16,32 @@ Check Point ClougGuard provides Compliance, Vulnerability Assessment, Visibility
 To install the chart with the chosen release name (e.g. `my-release`), run:
 
 ```bash
-$ helm repo add checkpoint https://raw.githubusercontent.com/CheckPointSW/charts/master/repository/
-$ helm install my-release checkpoint/cp-resource-management --set-string credentials.user=[CloudGuard API Key] --set-string credentials.secret=[CloudGuard API Secret] --set-string clusterID=[Cluster ID] --namespace=[Namespace] --create-namespace
+$ helm repo add checkpoint-ea https://raw.githubusercontent.com/CheckPointSW/charts/ea/repository/
+$ helm install asset-mgmt checkpoint-ea/cp-resource-management --set-string credentials.user=[CloudGuard API Key] --set-string credentials.secret=[CloudGuard API Secret] --set-string clusterID=[Cluster ID] --namespace=[Namespace] --create-namespace
 ```
 
-These are the additional optional flags for available add-ons:
+These are the additional optional flags for to enable add-ons:
 
 ```bash
 $ 
-$ --set addons.imageUploader.enabled 
+$ --set addons.imageUploader.enable=true 
 ```
 
-This command deploys a CloudGuard Resource Management agent.
+This command deploys a CloudGuard Resource Management agent as well as optional add-ons.
+
 
 > **Tip**: List all releases using `helm list`
 
+
+## Upgradeing the chart
+
+To upgrade the deployment and/or to add/remove additional feature run:
+
+```bash
+$ helm repo update
+$ helm upgrade asset-mgmt checkpoint/cp-resource-management --set addons.imageUploader.enable=[true/false] --set addons.flowLogs.enable=[true/false] --reuse-values --namespace=[Namespace]

```

Comment:
remove `--reuse-values `

You should mention that all enabled charts (and in fact all flags with non-default values) should be set in the upgrade command.